packages/agent/conf/demos/knb/babylon/oid4vci_metadata/knb_babylon.json

```diff
@@ -130,7 +130,7 @@
             ]
           }
         },
-        "id": "knb-bank",
+        "id": "BabylonBank2023",
```

Comment:
i think niels mentioned that we should not add any years. just checking that here

Reply:
fixed